File: programmers/even-number-and-odd-number.cpp
#include <iostream>
#include "solutions.h"

using namespace std;

void even_number_and_odd_number(){
    int number;
    cin >> number;

    if(number % 2 == 0){
        cout << "Even";
    }else {
        cout << "Odd";
    }
}

File: javascript-problem-solving/programmers/even-number-and-odd-number.js
process.stdin.setEncoding("utf8");
process.stdin.on("data", (data) => {
  const num = Number(data);

  if (num % 2 == 0) {
    console.log("Even");
  } else {
    console.log("Odd");
  }

  process.exit();
});
